scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -225,6 +229,83 @@ def __init__(self, test: dict, config: dict, definitions: SpecDefinitions, pics_
                     continue
                 get_constraints(value['constraints'])
 
+    def _update_mappings(self, test: dict, definitions: SpecDefinitions):
+        cluster_name = self.cluster
+        if definitions is None or not definitions.has_cluster_by_name(cluster_name):
+            self.argument_mapping = None
+            self.response_mapping = None
+            self.response_mapping_name = None
+            return
+
+        argument_mapping = None
+        response_mapping = None
+        response_mapping_name = None
+
+        if self.is_attribute:
+            attribute_name = self.attribute
+            if not (attribute := definitions.get_attribute_by_name(
```

Comment:
`:=` will require python 3.8 and above. Should we maybe split the assigment and if condition?

Reply:
I have splitted the assignment and the condition. Sorry for the noise.